files/index.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
Tenta mudar para pt-br... senão pode ter problemas com acentuação.
```suggestion
<html lang="pt-br">
```